rag/multimedia_rag/audio_transcriper.py

```
@@ -0,0 +1,188 @@
+
+class AudioRAG:
+    def __init__(self, workspace='slack_audio_workspace'):
+        try:
+            pxt.drop_dir(workspace, force=True)
+            pxt.create_dir(workspace)
+        except Exception as e:
+            logger.warning(f"Workspace setup issue: {e}")
+            # Try to use existing workspace
+            try:
+                pxt.get_dir(workspace)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)